test/distributed/test_c10d_nccl.py

```diff
@@ -3940,12 +3943,22 @@ def test_send_recv_object_list_subgroup(self, set_device: SetDeviceMethod):
             device = torch.device("cuda:%d" % self.rank)
         if self.rank == 0 or self.rank == 2:
             x = [{}]
-            c10d.recv_object_list(x, src=self.rank + 1, group=subgroup, device=device)
+            if group_rank:
+                c10d.recv_object_list(x, group_src=1, group=subgroup, device=device)
```

Comment:
instead of `group_src=1` is there a way to do something like `subgroup.local_rank() + 1`?

Reply:
yea, it would be equivalent. I just left it this way bc the test already hardcodes 4 ranks and 2 subgroups, so this is pretty clear IMO